plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version3/class-wc-rest-paypal-buttons-controller-test.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+declare(strict_types=1);
+
+use Automattic\WooCommerce\Enums\OrderStatus;
+
+/**
+ * class WC_REST_Paypal_Buttons_Controller_Test.
+ * PayPal Buttons Controller tests for V3 REST API.
+ */
+class WC_REST_Paypal_Buttons_Controller_Test extends WC_REST_Unit_Test_Case {
+	/**
+	 * Setup our test server, endpoints, and user info.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		// Mock Jetpack options to return a valid site ID.
+		add_filter(
+			'pre_option_jetpack_options',
+			function () {
+				return array( 'id' => 12345 );
+			}
+		);
+
+		// Return a Jetpack blog token.
+		add_filter(
+			'pre_option_jetpack_private_options',
+			function () {
+				return array( 'blog_token' => 'IAM.AJETPACKBLOGTOKEN' );
+			}
+		);
+
+		$this->endpoint = new WC_REST_Paypal_Buttons_Controller();
+		$this->user     = $this->factory->user->create(
+			array(
+				'role' => 'customer',
+			)
+		);
+		wp_set_current_user( $this->user );
+	}
+
+	/**
+	 * Tear down the test environment.
+	 */
+	public function tearDown(): void {
+		parent::tearDown();
+
+		remove_filter(
+			'pre_option_jetpack_options',
+			function () {
+				return array( 'id' => 12345 );
+			}
+		);
+		remove_filter(
+			'pre_option_jetpack_private_options',
```

Comment:
Same suggestion as above.

Reply:
Updated it in https://github.com/woocommerce/woocommerce/pull/61359/commits/f8a1be58a243e7c297e2596ecaf1c15daa8811db as well